test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -3349,92 +3131,74 @@ static Object checkFieldAccessMHOfAssortedType(MethodHandle[][] fieldAccessMHs,
 			case "LPoint2D;":
 				checkEqualPoint2D(fieldAccessMHs[i][0].invoke(instance), defaultPointPositions1);
 				Object pointNew = createPoint2D(defaultPointPositionsNew);
-				if (ifValue) {
-					instance = fieldAccessMHs[i][1].invoke(instance, pointNew);
-				} else {
+				if (!ifValue) {
```

Comment:
In the case `ifValue` is true, is the `fieldAccessMHs` always getters only ? If yes, please add a comment before this function to explicitly mention that.

Reply:
Yes. I have added a comment.